### PR TITLE
Attempting a cleanup / reorg

### PR DESCRIPTION
**Pull request by expede**
📜 Preview

### CODE REVIEW

**Comment by expede** [comment]
What is wrong with @cla-bot? I'm signed in the `.clabot` config!

**Comment by expede** [comment]
Ooooh the "actions user" 🤦‍♀️ That's annoying

File: README.md [hunk 1]
```diff
@@ -156,393 +156,982 @@ The executor MUST be the UCAN delegate. Their DID MUST be set the in `aud` field
...
+
+The `inv` field MUST include a link to the Task that the Receipt is for.
+
+### 9.1.2 Output
+
+The `out` field MUST contain the output of steps of the call graph, indexed by the task name inside the invocation. If the invocation is the implicit `"*"`, then the base64 hash of the concatenation of the URI, Ability and extensional fields MUST be used.
+
+The `out` field MAY omit any tasks that have not yet completed, or results which are not public. An `Task` may be associated to zero or more `Receipts`.
+
+A `Result` MAY include recursive `Receipt` CIDs in on the `Success` branch. As a Task may require subdelegation, the OPTIONAL `rec` field MAY be used to include recursive `Receipt`s.
```
Comment (simonwo) [inline README.md:964]: Perfect! Was going to make a comment to this effect on the previous version but you are way ahead as ever.

Why the base64 hash? Why not just the `Task` CID, or even better, the "memoization CID" ignoring the meta fields?

Comment (expede) [reply README.md:964]: @simonwo, ah indeed I need to update this to use the Closure CID now that it's a thing. Will fix 👍

File: README.md [hunk 2]
```diff
@@ -156,393 +156,982 @@ The executor MUST be the UCAN delegate. Their DID MUST be set the in `aud` field
...
+
+**NB: a Receipt this does not guarantee correctness of the result!** The statement's veracity MUST be only understood as an attestation from the executor.
+
+Receipts MUST use the same version as the invocation that they contain.
+
+## 9.1 Fields
+
+``` ipldsch
+type Receipt struct {
+  ran  Pointer
```
[the remark this answers is not in the document]
Comment (expede) [reply README.md]: Indeed, thanks! Fixed 👍

**Pull request merged**